test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Alias.Tests.ps1

```diff
@@ -46,17 +46,45 @@ Describe "Import-Alias DRT Unit Tests" -Tags "CI" {
 Describe "Import-Alias" -Tags "CI" {
 	$newLine=[Environment]::NewLine
 	$testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ImportAliasTestDirectory
-    $testAliases        = "pesteralias.txt"
-    $pesteraliasfile       = Join-Path -Path $testAliasDirectory -ChildPath $testAliases
+	$testAliases        = "pesteralias.txt"
+	$aliasFilenameMoreThanFourValues        = "aliasFileMoreThanFourValues.txt"
+	$aliasFilenameLessThanFourValues        = "aliasFileLessThanFourValues.txt"
+
+	$difficultToParseString_1		= '"abc""def"'
+	$difficultToParseString_2		= '"aaa"'
+	$difficultToParseString_3		= '"a,b"'
+    $pesteraliasfile    = Join-Path -Path $testAliasDirectory -ChildPath $testAliases
+    $aliasPathMoreThanFourValues    = Join-Path -Path $testAliasDirectory -ChildPath $aliasFileNameMoreThanFourValues
+    $aliasPathLessThanFourValues    = Join-Path -Path $testAliasDirectory -ChildPath $aliasFileNameLessThanFourValues
 
 	BeforeEach {
```

Comment:
We don't modify the files. So it must be `BeforeAll`. And lines 49-58 must be in the `BeforeAll` too.